README.md

```diff
@@ -1,3 +1,30 @@
-# OrbitDB-logotype
-Orbit's brand files, formats and guidelines
-This logotype and files have a CC-BY-SA License, so feel free to use it and modify it.
+# Logo
+
+Orbit's brand files, formats and guidelines.
+
+- [Guidelines](orbit_db_guidelines.pdf)
+
+### Standard Logos
+<img style="width:400px; margin:50px;" src="orbit_bd_logo_color.png" />
```

Comment:
Looking good generally, but t seems that the width property is not pickup and the logos appear *huge*. Prolly need to set the html-tag attribute `width` instead of defining them in CSS, eg. `<img width="400" src="orbit_bd_logo_color.png" />`.

Reply:
I noticed that but thought that GH Markdown would `max-width: 100%` them, maybe I'm wrong.

Reply:
Bizarre. My local renderer worked, looks like GitHub doesn't honor this. Changed.